reliability/circuitbreaker/breaker.go

```diff
@@ -51,11 +51,11 @@ func breakerCounterInc(name string, st status) {
 
 // Setting definition.
 type Setting struct {
-	// The threshold for the circuit to open.
+	// The failure threshold for the circuit to open.
```

Comment:
_Not strong on this at all._

Should we use the strict commenting convention of including the name of the field we are commenting?

Does this affect godoc? I actually thought that this was a requirement.

for example 
```suggestion
	// FailureThreshold represents the failure threshold for the circuit to open.
```

Reply:
Nice catch!